README.md

```diff
@@ -1,39 +1,80 @@
-# lambda-template
-A GitHub template for quickly starting a new AWS lambda project.
+# lambda-finops-s3-cost-report
 
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-macro-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
+An AWS Lambda for emailing monthly S3 service totals for the current account.
+
+## Design
+
+This lambda will query Cost Explorer for monthly service totals and S3 usage
```

Comment:
It's not clear to me why `s3` is in the name of this lambda.  Also what does `S3 usage type totals` mean? why do you need to query both cost explorer and s3 usage?   Can you please add a little more info to the docs to clarify?

Reply:
ok, so will you not be adding more info to the docs as i requested?